plugin/src/main/groovy/com/novoda/gradle/command/DemoModeTask.groovy

```diff
@@ -0,0 +1,64 @@
+package com.novoda.gradle.command
+
+import org.gradle.api.NamedDomainObjectContainer
+import org.gradle.api.tasks.TaskAction
+
+class DemoModeTask extends AdbTask {
+
+    boolean enable
+    NamedDomainObjectContainer<DemoModeExtension> commands
+
+    DemoModeTask() {
+        this.group = AndroidCommandPlugin.TASK_GROUP
+        this.description = "${enable ? 'Enables' : 'Disables'} demo mode on the connected device"
```

Comment:
I wouldn't emphasise `connected` here, we choose the device via the default means of `AdbTask`, right?

Reply:
Addressed while separating tasks. https://github.com/novoda/gradle-android-command-plugin/pull/125/commits/d2690644917ef57bfb914435c3633e923927f3d3